app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -15,19 +15,18 @@ def github
   def failure
     error = request.env["omniauth.error"]
 
-    Honeybadger.notify(
-      "Authentication failure",
-      error_message: error&.message,
-      error_class: error.to_s,
-      context: {
-        error_reason: error&.error_reason,
-        error_type: error&.error,
-        error_uri: error&.error_uri,
-        provider: request.env["omniauth.strategy"].name,
-        origin: request.env["omniauth.strategy.origin"],
-        params: request.env["omniauth.params"],
-        cookie: request.env["rack.request.cookie_hash"]
-      },
+    DatadogStatsClient.increment(
+      "auth.failure",
```

Comment:
Not blocking but should we name this metric `omniauth.failure` so if we end up sending metrics for other types of authentication we can differentiate?